frontend/src/pages/HomePage.tsx

```diff
@@ -18,25 +26,33 @@ export default function HomePage () {
         })();
     }, []);
 
+    const deleteHandler = (id: string) => {
+        props.deleteBook(id);
+    }
+    const updateHandler = (book: Book) => {
+        props.updateBook(book);
+    }
+
     return (
         <>
             <h1>HomePage</h1>
             <p>List of your Books</p>
-            {books.map((book) =>
-                <ul key={book.id}>
+            {props.books.map((book) =>
+                <ul key={props.book.id}>
                     <li>
                         <h2>{book.title}</h2>
                         <p>{book.author}</p>
                         <p>{book.description}</p>
-                        <img src={book.imageUrl} alt={book.title}/>
+                        <img src={book.imageUrl} alt={props.book.title}/>
```

Comment:
Mir ist nicht ganz klar, warum du in deinen props ein `book` hast. 
Du hast doch schon `books` und gehst hier über jedes einzelne Element in `books` drüber. Dieses Element solltest du hier doch referenzieren und nich irgendein anderes, was aus den props kommt?